packages/auth/amplify_auth_cognito_android/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AuthCognito.kt

```diff
@@ -165,6 +167,57 @@ open class AuthCognito :
         return applicationContext!!.packageName
     }
 
+    /**
+     * Fetches the legacy credentials set by the Android SDK
+     */
+    override fun getLegacyCredentials(identityPoolId: String, appClientId: String, result: NativeAuthPluginBindings.Result<NativeAuthPluginBindings.LegacyCredentialStoreData>) {
+        val legacyUserPoolStore = getLegacyKeyValueStore("CognitoIdentityProviderCache")
```

Comment:
I liked the idea of having a link for reference to the iOS code. Can we do the same here just so we know where the values came from?